css-containment/Overview.bs

```diff
@@ -138,6 +138,37 @@ Layout Containment</h3>
 
 	2. The element must be a <a>formatting context</a>.
 
+	3. If a <a>fragmentation context</a> participates in <a>layout containment</a>,
+		the first element with <a>layout containment</a>
+		affecting the <a>fragmentation context</a>
+		must “trap” the remainder of the <a>fragmented flow</a>.
+		<a>Fragmentation</a> must not continue past the <a>layout containment</a> boundary,
+		and the last <a>fragmentation container</a>
+		within the first <a>layout containment</a> boundary
+		is treated as if it is the last <a>fragmentation container</a>
+		in its <a>fragmentation context</a>.
+
+		If subsequent <a>fragmentation containers</a> in the <a>fragmentation context</a>
+		are only generated when more content remains in the <a>fragmented flow</a>,
+		then they are not generated.
+		If they would exist regardless,
+		they remain part of the <a>fragmentation context</a>,
+		but do no receive any content from the <a>fragmented flow</a>.
+
+		Specifically:
+			- <a spec=css-regions>CSS Regions</a> following the one which traps the content
```

Comment:
The linking database is managed by Shepherd, and it parses all the CSS specs, not just those produced by Bikeshed (though the other specs may or may not have their <dfn>s marked up to be recognized properly). The spec database for drafts is updated within minutes after a push. Specs on /TR get updated at least once per day (midnight pacific time).